File: CS/1929_Concatenation_of_Array.cs
public class Solution {
    public int[] GetConcatenation(int[] nums) {
        int[] result = new int[2*nums.Length];
        for(int i = 0; i < nums.Length; i++){
            result[i] = nums[i];
            result[i+nums.Length] = nums[i];
        }
        return result;
    }
}

File: Cpp/1929_Concatenation_of_Array.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> result(nums);
        for(int i = 0; i < nums.size(); i++){
            result.push_back(nums[i]);
        }
        return result;
    }
};
